docs/runbooks/KubeMacPoolDown.md

```diff
@@ -0,0 +1,24 @@
+# KubeMacPoolDown
+
+## Meaning
+
+KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.  
+
+## Impact
+
+If KubeMacPool-manager pod is down then virtualMachine objects cannot be created.
```

Comment:
`virtualMachine objects cannot be created.` Is it correct? Can't I create a VirtualMachine object? Or can't I start it? Does it happen in VirtualMachine level or VirtualMachineInstance level or virt-launcher level? Where will we observe the issue?

Reply:
@RamLavi ^^

Reply:
In short - yes. this is why this alert is critical.
In more detail - KMP has a webhook on `VirtualMachine` (not `VirtualMachineInstance`, which are running `VirtualMachine`s). When a `VirtualMachine` is created/updated, all the related webhooks are called.

this means that if the kmp-manager pod is down, the webhook (and as  result the `VirtualMachine` creation) will hang until KMP is up again and the webook is called.

This is exactly why this alert is so important.

Reply:
@erkanerol ^^

Reply:
@RamLavi  thanks for the detailed explanation.